packages/react/src/components/Comments/EmojiMartPicker.tsx

```diff
@@ -0,0 +1,23 @@
+// From https://github.com/missive/emoji-mart/blob/main/packages/emoji-mart-react/react.tsx
+import React, { useEffect, useRef } from "react";
+import { Picker } from "emoji-mart";
```

Comment:
Does this include the whole bundle of emoji mart data? Or is that separate?

Do we need this as a dynamic import?

Reply:
Don't think so, iirc from doing manual testing the emojis are fetched on initial render. Diving into the source code seems to support this:

https://github.com/missive/emoji-mart/blob/main/packages/emoji-mart/src/components/Picker/Picker.tsx#L112

The `_init` gets called in the `Picker` component's `componentWillReceiveProps` function, so either way it's not until the first render.

Reply:
Alright, cool, wasn't aware of how it works